eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -26,6 +26,64 @@ steps:
     inlineScript: |
       Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
 
+- task: CredScan@2
+  inputs:
+    toolMajorVersion: "V2"
+
+- task: SdtReport@1
+  displayName: "Generate Analysis Report"
+  inputs:
+    CredScan: true
+    ToolLogsNotFoundAction: "Standard"
+
+- task: TSAUpload@1
```

Comment:
I think `TSAUpload task` is not required for PR builds. We should fail the build if issues are detected.

Reply:
Thank you for your review. How do I fail the build from `CredScan@2` task after `SdtReport@1` is published with `PublishSecurityAnalysisLogs`.  If I just use `continueOnError: "false` with CredScan then it can't publish the report.